releasenotes/notes/feat-serverless-scheduler-31a819bc9eb4f332.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds serverless_scheduler, which flushes data after 60 calls to `periodic`, which occur every second.
```

Comment:
I'd probably write something more high-level, this is not going to help user figuring out what's happening.
If there's no way to use it for an user, might not be worth a changelog entry. :)